games/o.yaml

```
@@ -684,19 +684,22 @@
   type: remake
   originals:
   - Diablo II
-  repo: https://github.com/essial/OpenDiablo2
+  repo: https://github.com/OpenDiablo2/OpenDiablo2
   url: https://opendiablo2.com/
-  development: sporadic
+  development: active
   status: semi-playable
   lang:
-  - C++
+  - Go
   framework:
   - SDL2
```

```suggestion
  - Ebiten
```
https://github.com/hajimehoshi/ebiten